RELEASENOTES/v2.0.0.rc-2.MD

```diff
@@ -0,0 +1,68 @@
+- [Notary v2 Specs](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-rc.2)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-rc.2)*** (Updated in this release)
+
+## Goal of the release
+- Notary v2 specifications
+    - Updates to support signing and verification using the fallback method for referrers, as specified in the 1.1 version release candidate of the OCI Distribution specification. With this method, Notary v2 now supports a reference types solution but within the bounds of OCI 1.0 Image and Distribution specs 
+ - Update "Notation" (CLI client) implementation to match the latest specifications. With this new version of Notation client, users can
+    - Choose the method to manage signatures inside their registries - image manifest or artifact manifest.
+    - Provide optional meta-data at time of signing as key-value pairs.
+    - Use a new "inspect" command to see relevant details encoded in all signtures associated with an image.
+    - Use a new "completion" command to generate the corresponding completion script for their shells. 
+  
+  - Updated libraries for Notation for
+    - Improved error messaging, documentation, dependencies and bug fixes
+
+    
+*This is the second supported release from the Notary v2 community. Refer prior (RC-1) release description [here](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.rc-1.MD) and release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md)
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in registries supporting signatures as references to images.
+
+In this release of Notation command, the following CLI commands are supported. 
+  - **sign**        Sign OCI artifacts
+  - **verify**      Verify OCI artifacts
+  - **certificate** Manage certificates in trust store for verifying 
+  - **key**         Manage keys used for signing
+  - **list**        List signatures of the signed artifact
+  - **login**       Login to registry
+  - **logout**      Log out from the logged in registries
+  - **plugin**      Manage plugins  
+  - **version**     Show the notation version information
+  - **help**        Help about any command
+  *The following two new commands were added in this RC-2 release*
+  - **inspect**     Inspect all signatures associated with the signed artifact 
+  - **completion**  Generate the autocompletion script for the specified shell
```

Comment:
If this is from the CLI framework, I suggest not mentioning it.

Reply:
It from CLI framework, and it shows up in the Notation -- help. Hence, we decided to add a line in there.